### PR TITLE
feat: upgrade schema.org URLs to https

### PR DESCRIPTION
**Pull request by ibousfiha**
| Q             | A
| ------------- | ---
| Branch?       | master 
| Tickets       |  
| License       | MIT
| Doc PR        | api-platform/docs#... 
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
<!--

Hi, this is a simple upgrade of schema.org URLs to https

### CODE REVIEW

**Comment by dunglas** [comment]
The schema.org namespace is `http://`, not `https://`. This cannot be merged, this would break interoperability with all other RDF tools aware of Schema.org.

See https://stackoverflow.com/a/27112323

**Comment by dunglas** [comment]
Actually, it looks like Schema.org is in the process of supporting both schemes: https://github.com/schemaorg/schemaorg/issues/2853

So we may switch, but we must double-check that it doesn't break anything relying on Schema.org on other repositories (core and admin especially).

**Comment by ibousfiha** [comment]
Sure, please let me know if I should correct anything else or to do further tests.

**Review by UnseenBook — CHANGES_REQUESTED** [review]
This week I tried moving to https://schema.org as well, but I encountered a problem with the cardinality config. In the `v1.owl` file all links are to http://schema.org and I couldn't find an alternative.

**Comment by ibousfiha** [comment]
Needs to check phpunit tests

File: src/TypesGeneratorConfiguration.php [hunk 1]
```diff
@@ -30,7 +30,7 @@ final class TypesGeneratorConfiguration implements ConfigurationInterface
 {
     public const SCHEMA_ORG_URI = 'https://schema.org/version/latest/schemaorg-current-http.rdf';
     public const GOOD_RELATIONS_URI = 'https://archive.org/services/purl/goodrelations/v1.owl';
```
[the remark this answers is not in the document]
Comment (alanpoulain) [reply src/TypesGeneratorConfiguration.php:32]: @dunglas what do you think of having the HTTPS version hosted on the API Platform website?

**Comment by alanpoulain** [comment]
I've fixed the tests, thank you @ibousfiha!

**Pull request merged**